src/utils.js

```diff
@@ -16,11 +16,11 @@ module.exports.deferred = function () {
 };
 
 /**
- * Creates promise that will resolve after the given time (in ms).
+ * Returns a default logger that does nothing.
  */
 
-module.exports.timeout = function (ms, returnValue) {
-  return new Promise(resolve => setTimeout(() => resolve(returnValue), ms));
+module.exports.getDefaultLogger = function () {
+  return { error: () => {}, info: () => {}, warn: () => {} };
```

Comment:
Should it use the `console.xxx()` methods?

Reply:
This package needs no logging to do what it does. Since it is an extra, I would prefer not to define anything and let the user decide what to do.